packages/mui-icons-material/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/icons-material",
-  "version": "5.2.1",
+  "version": "5.2.2",
```

Comment:
@mnajdova @michaldudak Should this also be `5.2.4`?

Reply:
Yep